man/systemd-coredump.xml

```diff
@@ -58,59 +58,77 @@
-    <citerefentry><refentrytitle>coredump.conf</refentrytitle><manvolnum>5</manvolnum></citerefentry>
-    for details. By default, <command>systemd-coredump</command> will log the coredump including a
-    backtrace if possible, and store the core (contents of process' memory contents) in an external
-    file on disk in <filename>/var/lib/systemd/coredump</filename>.</para>
+    <para>By default, <command>systemd-coredump</command> will log the core dump including a backtrace
+    if possible to the journal and store the core dump itself in an external file in
+    <filename>/var/lib/systemd/coredump</filename>.</para>
 
-    <para>When the kernel invokes <command>systemd-coredump</command> to handle a coredump,
-    it will connect to the socket created by the <filename>systemd-coredump.socket</filename>
+    <para>When the kernel invokes <command>systemd-coredump</command> to handle a core dump,
+    it will connect to a socket created by the <filename>systemd-coredump.socket</filename>
     unit, which in turn will spawn a <filename>systemd-coredump@.service</filename> instance
-    to process the coredump. Hence <filename>systemd-coredump.socket</filename>
+    to process the core dump. Hence <filename>systemd-coredump.socket</filename>
     and <filename>systemd-coredump@.service</filename> are helper units which do the actual
-    processing of coredumps and are subject to normal service management.</para>
+    processing of core dumps and are subject to normal service management.</para>
 
-    <para>The log entry and a backtrace are stored in the journal, and can be viewed with
-    <citerefentry><refentrytitle>journalctl</refentrytitle><manvolnum>1</manvolnum></citerefentry>.
-    <citerefentry><refentrytitle>coredumpctl</refentrytitle><manvolnum>1</manvolnum></citerefentry>
-    may be used to list and extract coredumps or load them in
-    <citerefentry project='man-pages'><refentrytitle>gdb</refentrytitle><manvolnum>1</manvolnum></citerefentry>.
+    <para>The behavior of a specific program upon reception of a signal is governed by a few
+    factors which are described in detail in
+    <citerefentry project='man-pages'><refentrytitle>core</refentrytitle><manvolnum>5</manvolnum></citerefentry>.
+    In particular, the core dump will only be processed when the related resource limits are sufficient.
+    </para>
+  </refsect1>
+
+  <refsect1>
+    <title>Configuration</title>
+    <para>For programs started by <command>systemd</command> process ressource limits can be set by directive
+    <varname>LimitCore=</varname>, see
+    <citerefentry><refentrytitle>systemd.exec</refentrytitle><manvolnum>5</manvolnum></citerefentry>.
     </para>
 
-    <para>The coredump helper is invoked anew each time. Therefore, any configuration
-    changes will take effect on the invocation of <command>systemd-coredump</command>.
+    <para>In order to take action <command>systemd-coredump</command> must be set as value of
```

Comment:
"In order to be used \<command>systemd-coredump\</command> must be configured in `<citerefentry project='man-pages'><refentrytitle>sysctl</refentrytitle><manvolnum>8</manvolnum></citerefentry>` parameter `<varname>kernel.core_pattern</varname>`. The syntax of this parameter is explained in"